safehttp/safehttptest/recorder.go

```diff
@@ -17,7 +17,7 @@ package safehttptest
 import (
 	"encoding/json"
 	"errors"
-	"html/template"
+	"github.com/google/safehtml/template"
```

Comment:
Why is this usage replaced? Was there a linter complaining about `html/template`?

Reply:
Yep.